### DISCUSSION TITLE
Hisense Auto-resets Changes

### DISCUSSION THREAD

**Discussion by pburke90** [PredatH0r/ChanSort#319] [Q&A]
I have a Hisense B7500 and have placed the channels into an order I want, some numbers skipped, and have also changed some channel names. I find that after the new list is uploaded to the TV, soon after the channels are tuned in, some channels names revert and all channels bunch together instead of remaining in the separate number groups I put them into. Is there a way of stopping the TV from auto-undoing the changes I've made? There is an 'auto-update channels' setting in the menu but I have always had that deactivated.

**Comment by PredatH0r** [comment]
Can you send me your list please (the unmodified .bak + the modified file)?

Did you select a specific service provider (Astra, Sky, ...) during the channel search? I generally recommend to use a provider neutral list whenever possible, because many TVs fail to work properly when importing a list over a preset in memory.

**Comment by pburke90** [comment]
I will get the files for you and send them over soon. The TV asks to pick what sort of search is to be performed (terrestrial / satellite / cable or a combination of these) but that's it, nothing specific like Sky/Virgin etc. so it is set up to search terrestrial and two satellites controlled by Diseqc but no providers specific channel list has been applied (to my knowledge).

**Comment by pburke90** [comment]
Hi again, I found your e-mail on your profile and have sent the files as requested. Many thanks in advance.

**Comment by pburke90** [comment]
Hi, just wondering if you managed to find a solution for this? Appreciate it. :)

**Comment by PredatH0r** [comment]
I haven't found anything yet.
Do you have a setting on your TV similar to "automatically update channels"?
Some TVs have that for DVB-C/T only, but what you describe sounds very much like automatic update of the list based on data received over DVB.

I've been working through the backlog the last couple weeks, but there is still a lot more to do. I'll give your files a closer look in the next couple days.

**Comment by pburke90** [comment]
Just to follow up on this, I reset the TV to factory settings and chose a different country during setup. The TV tuned in all satellite and terrestrial channels. I used ChanSort and numbered and renamed them as required. The TV now leaves the numbers as I have selected, even with huge gaps between them, but for all channels, terrestrial and satellite, it reverts the name back to its original. This is despite having both auto update channels and update channel lists turned off. This is different than the last time when I left the country as UK and most channel names stayed the same but the numbers all closed together to remove any gaps. So it seems the settings are different for each country and I might have to just keep trying to find one that works?
